addusers.py

```diff
@@ -43,6 +43,7 @@
 #local
 import message
 import spreadsheet
+from moc_utils import get_absolute_path
```

Comment:
(not related to this, but) This reminds me: Do we have a `requirements.txt` that can be executed: `pip install -r requirements.txt` to install all the dependencies? 

I know this is not a python lib. But are we using any?